support.js

```diff
@@ -53,9 +57,26 @@ if (Cypress.env('coverage') === false) {
     windowCoverageObjects.forEach(cover => {
       sendCoverage(cover.coverage, cover.pathname)
     })
+
+    if (!hasE2ECoverage()) {
+      if (hasUnitTestCoverage()) {
+        cy.log(`👉 Only found unit test code coverage.`)
```

Comment:
I think we should make these logs more specific to show that this logging is coming from the code-coverage plugin. People put a LOT of stuff in their repos and I'm afraid they'll see this random log and assume it's coming directly from Cypress somewhere. Thoughts to make this clearer:

- Have this be like, a custom command that logs the command name as 'CODE COVERAGE'? 
- At the end of log say something like 'logged from [code-coverage](https://github.com/cypress-io/code-coverage)